routes/socket/models.js

```diff
@@ -163,8 +163,8 @@ module.exports.formattedUserList = isAEM => {
 			// blacklist: user.blacklist,
 			customCardback: user.customCardback,
 			customCardbackUid: user.customCardbackUid,
-			eloOverall: user.eloOverall ? Math.floor(user.eloOverall) : undefined,
-			eloSeason: user.eloSeason ? Math.floor(user.eloSeason) : undefined,
+			eloOverall: (user.staffDisableVisibleElo && user.eloOverall) ? Math.floor(user.eloOverall) : undefined,

```

Comment:
uh this ternary makes no sense? this only shows the Elo for mods who are hiding their Elo lol